modules/protocol/src/main/scala/Domains.scala

```diff
@@ -102,6 +102,7 @@ object ConsumerTag extends TaggedOpaqueComp[ShortString, ConsumerTag] {
   inline def of(inline t: String): ConsumerTag = ShortString(t)
   def empty: ConsumerTag = ShortString.empty
   def from(t: String): Either[String, ConsumerTag] = ShortString.from(t)
+  def random: ConsumerTag = ShortString.from(UUID.randomUUID)
```

Comment:
Creating a UUID is a side-effect and shouldn't be in a pure function, it also requires external dependencies to make it cross platform. The cats-effect way of doing it in a cross-platform way without requiring external dependencies is by using `UUIDGen[F]`.

I think it's better to leave it to be a user side decision, as there are no restrictions regarding what's the value of the ctag.

Reply:
Ok I'll remove UUID and get back the `consumer-x` as ctag default.